shared/fs/common/path-item-action/layout.js

```diff
@@ -95,7 +95,7 @@ const getRawLayout = (
             }
           : {}),
         copyPath: true,
-        delete: pathItem.type === 'file' || flags.enableDeleteFolder,
+        delete: (pathItem.writable && pathItem.type === 'file') || flags.enableDeleteFolder,
```

Comment:
Seems like pathItem.writable should be outside the parentheses and the flag should be inside